app/infrastructure/persistence/repositories/document_repo.py

```
@@ -132,12 +148,13 @@ async def mark_parent_not_latest_if_current(
         return result.rowcount == 1
 
     async def update(self, document: DocumentResult) -> DocumentResult:
-        """Update document from DTO (e.g. is_latest_version); return updated DTO."""
+        """Update document from DTO (e.g. is_latest_version, document_type); return updated DTO."""
         orm = await self._get_orm_by_id(document.id)
         if not orm:
             raise ValueError(f"Document {document.id} not found")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)